mysql_servers.json

```diff
@@ -235,6 +219,23 @@
       "script": "_scripts/firestore.sh",
       "max_restarts": "1",
       "min_uptime": "2m"
+    },
+
+    {
+      "name": "MySQL server PORT 3306",
+      "script": "_scripts/mysql.sh",
+      "max_restarts": "1",
+      "min_uptime": "2m"
+    },
+    {
+      "name": "auth-server db mysql PORT 8000",
```

Comment:
Mind moving this one below mailer helper, for my muscle memory of `./pm2 logs 1`?

Reply:
ah yes thanks